python/rpdk/java/codegen.py

```diff
@@ -304,6 +315,9 @@ def _init_settings(self, project):
         project.runtime = self.RUNTIME
         project.entrypoint = self.ENTRY_POINT.format(self.package_name)
         project.test_entrypoint = self.TEST_ENTRY_POINT.format(self.package_name)
+        project.executable_entrypoint = self.EXECUTABLE_ENTRY_POINT.format(
```

Comment:
We should also add the `executable_entrypoint` configuration in the `generate` command if it doesn't exist (but don't override it).